examples/opensora_hpcai/opensora/models/layers/blocks.py

```diff
@@ -711,12 +711,7 @@ def _get_cached_emb(
             grid_h *= base_size / h
             grid_w *= base_size / w
 
-        orig_dtype = grid_h.dtype
-        if orig_dtype == ms.bfloat16:  # BUG MS2.3rc1: ops.meshgrid() doesn't support bf16
-            grid_h = grid_h.astype(ms.float32)
-            grid_w = grid_w.astype(ms.float32)
-        grid_h, grid_w = ops.meshgrid(grid_w, grid_h, indexing="ij")  # here w goes first
-        grid_h, grid_w = grid_h.astype(orig_dtype), grid_w.astype(orig_dtype)
+        grid_h, grid_w = ms.numpy.meshgrid(grid_w, grid_h, indexing="ij")  # here w goes first
```

Comment:
not suggested to change.

1. `ms.numpy.meshgrid` is built based on a collection of operators, while `ops.meshgrid` uses ops.Meshgrid single operators. `ms.numpy.meshgrid` should have performance drops.
2. `ops.meshgrid` is verifed in Optimimzer Parallel and Sequence parallel based on the recent Mindspore fixe patch, but   `ms.numpy.meshgrid` is not checked.

Reply:
`ops.meshgrid` doesn't work in `PyNative` mode. Will remove it for now.